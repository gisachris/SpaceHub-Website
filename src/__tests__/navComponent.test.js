import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import { render } from '@testing-library/react';
import Navigation from '../components/navigation';

test('renders correctly', () => {
  const { container } = render(
    <Router>
      <Navigation />
    </Router>,
  );

  expect(container).toMatchSnapshot();
});
