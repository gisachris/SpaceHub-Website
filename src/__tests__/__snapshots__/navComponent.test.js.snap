// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <section
    class="headerSection"
  >
    <nav
      class="navbar"
    >
      <span
        class="logoArea"
      >
        <img
          alt="logo fot the header"
          src="logo.png"
        />
        <h1
          class="logoText"
        >
          Space Traveler's Hub
        </h1>
      </span>
      <ul
        class="linksHolder"
      >
        <li>
          <a
            class=""
            href="/rockets"
          >
            Rockets
          </a>
        </li>
        <li>
          <a
            class=""
            href="/missions"
          >
            Missions
          </a>
        </li>
        <li>
          <a
            class=""
            href="/profile"
          >
            MyProfile
          </a>
        </li>
      </ul>
    </nav>
  </section>
</div>
`;
